### PR TITLE
[PRG-2025] Add sponsor levels

### PR DESCRIPTION
**Pull request by EtssMM**
- Adding sponsor levels
- Fixing default city redirect

### CODE REVIEW

File: content/events/2025-prague/sponsor.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Type = "event"
 Description = "Sponsor DevOpsDays Prague 2025"
 +++
 
-We greatly value sponsors for this open event.  If you are interested in sponsoring, please drop us an email at [{{< email_organizers >}}].
+We greatly value sponsors for this open event.  If you are interested in sponsoring, please <a href="mailto:sponsors@devopsdays.cz?subject=Interested%20in%20Sponsoring%20DevOpsDays%20Prague%202025">send us an email</a>.
```
Comment (toshywoshy) [inline content/events/2025-prague/sponsor.md:7]: Is there a reason you're not using the city email of DoD?

**Review by toshywoshy — CHANGES_REQUESTED** [review]

File: content/events/2025-prague/sponsor.md (hunk 1 above)
Comment (EtssMM) [reply content/events/2025-prague/sponsor.md:7]: There are 2 reasons:
- We want to be able to respond from the same domain
- We want to have flexibility who is receiving the emails as we might shuffle volunteers here and there and don't want to update the org list every time.

Comment (toshywoshy) [reply content/events/2025-prague/sponsor.md:7]: Can someone from core approve the off DoD email with the above reasons?
@mattstratton @nimbinatus @phrawzty

Comment (EtssMM) [reply content/events/2025-prague/sponsor.md:7]: We have used the same last time and I have seen it on other cities as well. Is there any reason why we should avoid it?

Comment (toshywoshy) [reply content/events/2025-prague/sponsor.md:7]: Well, there is always a trust issue, the webdomain is devopdays.org and using prague[at]devopsdays[dot]org seems better
However as you indicated correctly there is presidence on using another email address then the DoD mailing list.

**Review by toshywoshy — APPROVED** [review]

**Pull request merged**